plugins/woocommerce/src/Admin/Features/OnboardingTasks/Tasks/Payments.php

```diff
@@ -97,4 +102,19 @@ function( $gateway ) {
 
 		return ! empty( $enabled_gateways );
 	}
+
+	/**
+	 * The task action URL.
+	 *
+	 * @return string
+	 */
+	public function get_action_url() {
+		// If the React-based Payments settings page is enabled, we want the task to link to the Payments Settings page.
+		if ( Features::is_enabled( 'reactify-classic-payments-settings' ) ) {
+			return admin_url( 'admin.php?page=wc-settings&tab=checkout' );
+		}
+
+		// Otherwise, we want the task behavior to remain unchanged (link to the Payments task page).
+		return '';
```

Comment:
a bit weird code-wise - returning empty string will remain the behaviour unchanged, but it works 🤷

Reply:
Yeah. That is how the tasks work.